docs/dashboards/entity-dashboard.asciidoc

```diff
@@ -0,0 +1,94 @@
+[[detection-entity-dashboard]]
+= Entity dashboard
+
+The Entity Analytics dashboard provides a centralized view into emerging insider threats - including host risk, user risk, and notable anomalies from within your network. Use these analytics to triage emerging threats, investigate, and respond.
+
+
+.Requirements
+[sidebar]
+--
+The following is required to display the Entity Analytics dashboard: 
+
+* A https://www.elastic.co/pricing/[Platinum subscription] or higher.
+* To display host and user risk scores, the host risk score and user risk score features must be enabled. You can do this directly from the dashboard. For more information, refer to Enable host risk score and <<deploy-user-risk-score, Enable user risk score>>.
+* To display notable anomalies, the following machine learning jobs must be {ml-docs}/ml-ad-run-jobs.html[installed and run]: 
+** `auth_rare_source_ip_for_a_user`
+** `suspicious_login_activity`
+** `packetbeat_dns_tunneling`
+** `packetbeat_rare_server_domain`
+** `packetbeat_rare_dns_question`
+** `v3_windows_anomalous_script`
+--
+
+
+The dashboard includes the following sections:
+
+* Entity KPIs 
+* Host risk scores
+* User risk scores
+* Notable anomalies 
+
+
+[role="screenshot"]
+image::images/entity-dashboard.png[Entity dashboard]
+
+*Entity KPIs (key performance indicators)*
```

Comment:
Is there a reason why all the "headings" are plain bold text instead of actual headings? This really limits navigation, since there's no page TOC, no ability to cross-ref, etc.